rfcs/034-location_location_location/README.md

```diff
@@ -0,0 +1,133 @@
+# RFC 034: Modelling Locations in the Catalogue API
+
+As part of upcoming work to add journals and holdings to the Catalogue API, we need to expand the range of locations which are presented by the API.
+Additionally, we want to model locations in a format suitable for the requesting service.
+
+This RFC describes the types of location we want to model, and proposes how they will be returned in the catalogue API.
+
+**Current status:** this RFC outlines the problem, by describing the types of location we want to model.
+Please check if the information contained is accurate and complete.
+
+
+
+## The existing approach
+
+The Catalogue API contains *Works*, and Works contain *Items*.
+In turn, each Item can have *Locations*, which tell the user how they can get the item.
+We're not planning to change the Item model; we want to consider if the Location model is fit for purpose.
+
+<details>
+<summary>Examples of physical and digital locations in the existing model</summary>
+
+Physical location:
+
+```json
+{
+  "locationType": {
+    "id": "scmac",
+    "label": "Closed stores Arch. & MSS",
+    "type": "LocationType"
+  },
+  "label": "Closed stores Arch. & MSS",
+  "accessConditions": [
+    {
+      "status": {
+        "id": "open",
+        "label": "Open",
+        "type": "AccessStatus"
+      },
+      "terms": "The papers are available subject to the usual conditions of access to Archives and Manuscripts material.",
+      "type": "AccessCondition"
+    }
+  ],
+  "type": "PhysicalLocation"
+}
+```
+
+  Digital location:
```

Comment:
I think it's worth including a digitised item location here, instead or in addition to a Miro image. Ie, something that is linking to a IIIF Presentation API manifest.

Reply:
Added.